docs/marketplace-required-csv-annotations.md

```diff
@@ -0,0 +1,85 @@
+# Marketplace Required CSV Annotations
+
+An operator's CSV must contain the following annotations for it to be displayed properly within the Marketplace UI:
+
+```yaml
+metadata:
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: The operator's certification. A process to get this value will be defined at a later date.
+    description: |-
+      A short description of the operator that will be displayed on the marketplace tile.
+      If this annotation is not present, the `spec.description` value will be shown instead.
+      In either case, only the first 135 characters will appear.
+    containerImage: The repository that hosts the operator image.
```

Comment:
how complete does this need to be? Does it it need a hostname? Can we give an example, eg `registry.redhat.io/foo/bar:baz`?